blazor/gantt-chart/performance.md

```diff
@@ -0,0 +1,101 @@
+title: Performance tips for Blazor Gantt Chart Component | Syncfusion
+description: Check out and learn how to improve the loading performance of Blazor Gantt Chart Component even binding large data set.
+platform: Blazor
+control: Gantt Chart
+documentation: ug
+---
+
+# Performance tips for Blazor Gantt Chart Component
+
+This article is a comprehensive guide on enhancing the loading performance of the Blazor Gantt Chart component, particularly when managing large datasets and numerous columns. It provides valuable insights into effective strategies and best practices for binding extensive data sources without encountering performance degradation. Through detailed explanations and actionable tips, this resource aims to equip users with the knowledge needed to optimize the performance of the Gantt Chart component during data binding, ensuring a smooth and responsive user experience.
+
+## How to improve loading performance by binding large dataset
+
+The Blazor Gantt Chart renders rows, columns, and taskbars, creating a significant number of DOM elements, especially with large datasets. For instance, binding 10 rows and 10 columns generates 100 elements in the grid area and additional elements in the chart area. To optimize performance and reduce loading times, it is recommended to limit the number of rows and columns rendered at once. Implementing techniques such as virtualization, load on demand, and efficient data binding can help manage large datasets effectively, enhancing the overall user experience.
+
+### Optimizing performance with virtualization
+
+To enhance the efficiency of the Blazor Gantt Chart, particularly when handling large datasets, it is recommended to use [virtualization](https://blazor.syncfusion.com/documentation/gantt-chart/virtualization) techniques. These methods can significantly reduce the load on your application, improving overall performance.
+
+1.	**Row Virtualization**: [Row virtualization](https://blazor.syncfusion.com/documentation/gantt-chart/virtualization#row-virtualization) in the Blazor Gantt Chart allows the efficient handling and display of large datasets by rendering only the visible rows within the Gantt viewport, rather than loading the entire dataset simultaneously. This approach optimizes rendering and reduces the DOM size, resulting in faster load times and smoother interactions. This feature is particularly effective during vertical scrolling, as rows are loaded on-demand based on the current scroll position.
+
+2. **Column Virtualization**: [Column virtualization](https://blazor.syncfusion.com/documentation/gantt-chart/virtualization#column-virtualization) optimizes the rendering of Gantt Chart with a large number of columns by displaying only the columns currently within the viewport. This feature allows horizontal scrolling to reveal additional columns, significantly reducing the initial loading time and improving performance by limiting the number of DOM elements rendered at once.
```

Comment:
This feature render column cells dynamically during horizontal scrolling, significantly reducing the initial loading time and improving performance by limiting the number of DOM elements rendered at once.